tracer.spec

```diff
@@ -97,7 +85,7 @@ mkdir -p %{buildroot}/%{python3_sitelib}/tracer
 
 cp -a bin/tracer-2 %{buildroot}/%{_bindir}/
 cp -a bin/tracer-3 %{buildroot}/%{_bindir}/
-ln -s %{_bindir}/tracer-2 %{buildroot}/%{_bindir}/tracer
+ln -s %{_bindir}/tracer-3 %{buildroot}/%{_bindir}/tracer
```

Comment:
as I pointed on IRC that if binary provides same output on py2/py3 you should ship only one (python3) version.